idofront-serializers/src/main/kotlin/com/mineinabyss/idofront/serialization/SerializableItemStack.kt

```diff
@@ -51,6 +59,43 @@ data class SerializableItemStack(
      * [existing item][applyTo].
      */
     fun toItemStack(applyTo: ItemStack = ItemStack(type ?: Material.AIR)): ItemStack {
+
+        // Import ItemStack from Crucible
+        crucibleItem?.let { id ->
+            if (Plugins.isEnabled<MythicCrucible>()) {
+                MythicCrucible.core().itemManager.getItemStack(id)?.let {
+                    applyTo.type = it.type
+                    applyTo.itemMeta = it.itemMeta
+                } ?: logWarn("No Crucible item found with id $id")
+            } else {
+                logWarn("Tried to import Crucible item, but MythicCrucible was not enabled")
+            }
+        }
```

Comment:
I think the way forward to this is a proper extension system. For now this should be fine but I want to pollute the main toItemStack function as little as possible and avoid dependencies on other plugins in a broad module like this. I'll make an issue for it